### ISSUE TITLE
UX improvements to download older versions

### ISSUE THREAD

**Issue by openforfuture** [vector-im/element.io#67]
Please make easier to download older versions:

https://github.com/element-hq/element-desktop/issues/1505#issuecomment-1946421421

Thank you 🙏

**Comment by bnjbnj** [comment]
I agree, it would be helpful to have a linkt to the packages https://packages.element.io/desktop/ on the Download page from https://element.io/download as well as on the Github readme/Main page maybe? Thanks for developping this really great app, btw. :)
